sdk/azidentity/logging.go

```diff
@@ -6,20 +6,22 @@ package azidentity
 import (
 	"fmt"
 	"os"
+	"runtime"
 	"strings"
 
 	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
-	"github.com/Azure/azure-sdk-for-go/sdk/internal/runtime"
+	azlog "github.com/Azure/azure-sdk-for-go/sdk/azcore/log"
+	"github.com/Azure/azure-sdk-for-go/sdk/internal/log"
 )
 
 // LogCredential entries contain information about authentication.
 // This includes information like the names of environment variables
 // used when obtaining credentials and the type of credential used.
-const LogCredential azcore.LogClassification = "Credential"
+const LogCredential azlog.Classification = "Credential"
```

Comment:
You could also forego the need for `azlog` and just use `log.Classification` as they're the same type.